Enterprise-CMCS/cmcs-eregulations
# EREGCSC-2779 -- Vitest fixes

Resolves [EREGCSC-2779](https://jiraent.cms.gov/browse/EREGCSC-2779)

**Description**

The V4 refactor broke some front end tests and made some others obsolete.  This PR will remove the unneeded tests and fix the remaining broken tests.

**This pull request changes:**

- Adds new Makefile command to run Vitest UI locally
   - `make test.vitest.ui`
- Updates test fixtures to reflect updated API responses where necessary
- Removes unneeded assertions and test suites
- Removes failing snapshot tests and associated snapshot files
   - Keeps passing snapshot tests for now
- excludes MSW directory from coverage calculations

**Steps to manually verify this change:**

1. Green check marks
2. Maybe run the Vitest tests locally and see how it looks? 
   - `make test.vitest.ui`
